src/nav/root.yml

```diff
@@ -1,6 +1,7 @@
 title: root
 path: /
 pages:
+  - title: Welcome
```

Comment:
`nit` Do we also wanna add the "Full Stack Monitoring"? after the line break?

![Screen Shot 2022-10-03 at 3 01 35 PM](https://user-images.githubusercontent.com/26727669/193668718-61de569a-9afb-4f6c-b785-ec0161a38790.png)

Reply:
I'm gonna save that part for the ticket where we refactor the nav items themselves. I just added `welcome` so we could see the styling 💅